src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +103,53 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={query}
+              onChange={(e) => onQueryChange(e.target.value)}
```

Comment:
I'd recommend you create a separate function for this, that accepts the event (actually, you should just change the argument of `onQueryChange` function, as you do in the `onFormSubmit` function)
```suggestion
              onChange={onQueryChange}
```